src/core/include/openvino/op/softsign.hpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/util/unary_elementwise_arithmetic.hpp"
+
+namespace ov {
+namespace op {
+namespace v9 {
+class OPENVINO_API SoftSign : public util::UnaryElementwiseArithmetic {
```

Comment:
These is no `validate_and_infer_types() method`. Is `UnaryElementwiseArithmetic::validate_and_infer_elementwise_arithmetic()` sufficient?

Reply:
Consider additional check for integer types as this operation should allows to pass floating point only, 
and UnaryElementwiseArithmetic throws for boolean but not for integers.

Reply:
Added, I will test it out on Tuesday.